access/slack/README.md

```diff
@@ -69,7 +69,7 @@ spec:
     rules:
       - resources: ['access_request']
         verbs: ['list', 'read', 'update']
```

Comment:
The official guide has these in the allow
```
spec:
  allow:
    rules:
      - resources: ['access_request']
        verbs: ['list', 'read']
      - resources: ['access_plugin_data']
        verbs: ['update']
```
Which is one is more up-to-date? During my testing, it doesn't seem to make a difference.

Reply:
tldr: only list and read are required for `access_request`

Access plugins used to allow for the acceptance of an `access_request` as part of the plugin within the plugin's App.
Example in slack:
When an access request was created, slack plugin would send a message to a specific workspace/channel and users would be able to click on a Accept/Deny buttons.
That relaxed the security we were aiming for (not exactly sure about the full details).
So, now, we only send a message with a link to teleport web ui itself.